jobs/integr8ly/ocp4/osd/cluster/deprovision/Jenkinsfile

```diff
@@ -23,7 +23,48 @@ node("cirhos_rhel7") {
                     sh "ocm login --url=https://api.stage.openshift.com/ --token=${OCM_ACCESS_TOKEN}"
                 }
             } // stage
-            
+
+            stage("login to oc on BYOC clusters") {
+                script {
+                    if(params.BYOC == "true") {
+                        adminPassword = sh (
+                            script: "cat ${installConfigDirectory}${clusterCredentialsFileName} | jq -r .admin.password",
+                            returnStdout: true
+                        ).trim()
+                        clusterId = sh (
```

Comment:
there is `clusterID` declared above. You could reuse it here, e.g. replace with `clusterID`